### ISSUE TITLE
Change Hybrid input page names

### ISSUE THREAD

**Issue by cpaulgilman** [NREL/SAM#1570]
Change "Hybrid System" page to "Hybrid Summary" and "Hybrid System Costs" to "System Cost Summary."

However:

* "System" is more consistent with "System Design" pages for other technologies. Eventually there may be hybrid-specific inputs? See CSP System Design pages.
* We use "Summary" on the Results page, so using it for an input page could be confusing.

**Comment by janinefreeman** [comment]
I agree that those are potentially confusing. Let's brainstorm more and see if we can come up with something better...
